### PR TITLE
Adding the CMLE Custom Online Prediction Notebooks for Keras and PyTorch

### PR DESCRIPTION
**Pull request by ksalama**
- Added two notebooks to demonstrate the CMLE Custom Online Prediction code feature, via building and serving Keras and PyTorch Text Classifiers + Preprocessing

-  Adding the following in each Notebook:
   - Copyright
   - Overview
   - Dataset
   - Objective
   - Costs
- Added Colab Support
- Tested on JaaS

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/cloudml-samples/396)

### CODE REVIEW

File: notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb [hunk 1]
```diff
@@ -0,0 +1,1010 @@
...
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "WUEA9FKcy8fM"
+   },
+   "outputs": [],
+   "source": [
+    "MODEL_NAME='torch_text_classification'\n",
+    "VERSION_NAME='v201903'\n",
+    "RUNTIME_VERSION='1.12'\n",
```
Comment (gogasca) [inline notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: nit: Supported in TF version 1.13?

File: notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb [hunk 2]
```diff
@@ -0,0 +1,1010 @@
...
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "WbE2cKVE1PaX"
+   },
+   "outputs": [],
+   "source": [
+    "!gcloud alpha ml-engine versions create {VERSION_NAME} --model {MODEL_NAME} \\\n",
+    "--origin=gs://{BUCKET}/{MODEL_DIR}/ \\\n",
+    "--python-version=2.7 \\\n",
```
Comment (gogasca) [inline notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: Can we run this in Python 3.x?

File: notebooks/keras/Text Classification Using Keras and CMLE.ipynb [hunk 3]
```diff
@@ -0,0 +1,893 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "N0p05OnXRugd"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install tensorflow==1.12"
```
[the remark this answers is not in the document]
Comment (ksalama) [reply notebooks/keras/Text Classification Using Keras and CMLE.ipynb]: Done

File: notebooks/keras/Text Classification Using Keras and CMLE.ipynb [hunk 4]
```diff
@@ -0,0 +1,893 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "UWvrShlZjZwr"
+   },
+   "source": [
+    "# Text Classification Using Keras and Cloud ML Engine\n",
```
[the remark this answers is not in the document]
Comment (ksalama) [reply notebooks/keras/Text Classification Using Keras and CMLE.ipynb]: Done

File: notebooks/keras/Text Classification Using Keras and CMLE.ipynb [hunk 5]
```diff
@@ -0,0 +1,893 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "pU4D8prVtLNI"
+   },
+   "source": [
+    "## Model Deployment to CMLE"
```
[the remark this answers is not in the document]
Comment (ksalama) [reply notebooks/keras/Text Classification Using Keras and CMLE.ipynb]: Done

File: notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb (hunk 1 above)
Comment (ksalama) [reply notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: Done

File: notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb (hunk 2 above)
Comment (ksalama) [reply notebooks/pytorch/Text Classification Using PyTorch and CMLE.ipynb]: Done

File: notebooks/keras/Text Classification Using Keras and CMLE.ipynb [hunk 6]
```diff
@@ -0,0 +1,893 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "66JlKmfzvPhN"
+   },
+   "outputs": [],
```
[the remark this answers is not in the document]
Comment (ksalama) [reply notebooks/keras/Text Classification Using Keras and CMLE.ipynb]: Done

**Comment by ksalama** [comment]
I renamed Cloud ML Engine to AI Platform Serving. However, in the pricing part, since the GCP price calculator used Cloud ML Engine term, I kept it.

**Pull request merged**